src-opam/opam.ml

```diff
@@ -212,12 +210,13 @@ let opam_master_branch opam_master_hash =
     enable_0install_solver = true;
     with_vendored_deps = true;
     public_name = "opam-dev";
-    aliases = [ "opam-2.2" ];
-    (* TODO: Remove/update when opam 2.2 is branched *)
+    aliases = [];
```

Comment:
Assuming we still want to keep this field around even tho the todo is resolved, would it make sense, and help in the future, to add a comment on the `alias` field in the type definition (line 203) to note what this is meant to be used for?

Reply:
`aliases` is still used to link 2.0 to the default.  See L228.  The to do here referred to aliasing the `master` branch to `2.2` even though 2.2 was not released.